src/sentry/web/frontend/organization_integration_setup.py

```diff
@@ -20,46 +20,34 @@ class OrganizationIntegrationSetupView(ControlSiloOrganizationView):
     csrf_protect = False
 
     def handle(self, request: Request, organization, provider_id) -> HttpResponseBase:
-        try:
-            with sentry_sdk.configure_scope() as scope:
-                parent_span_id = scope.span.span_id
-                trace_id = scope.span.trace_id
-        except AttributeError:
-            parent_span_id = None
-            trace_id = None
+        with sentry_sdk.configure_scope() as scope:
+            scope.transaction.op = "integration.setup"
+            scope.transaction.name = f"integration.{provider_id}"
```

Comment:
please use this method and set the source route/view (both are fine). The source is important to tell ingest that the transaction name is 'low cardinality'.

https://github.com/getsentry/sentry-python/blob/e864eab559c2b37b44bdf6f353cbdb25c8f885ce/sentry_sdk/scope.py#L499-L501